api/keto/acl/v1/check_service.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package keto.acl.v1;
+
+import "keto/acl/v1/acl.proto";
+import "google/api/client.proto";
+
+option go_package = "github.com/ory/keto/api/keto/acl/v1;acl";
+option csharp_namespace = "Ory.Keto.Acl.V1";
+option java_multiple_files = true;
+option java_outer_classname = "CheckServiceProto";
+option java_package = "sh.ory.keto.acl.v1";
+option php_namespace = "Ory\\Keto\\Acl\\V1";
+
+// The service that performs authorization checks
+// based on the stored Access Control Lists.
+service CheckService {
+  option (google.api.default_host) = "keto.exampleapis.com";
+  // Check performs an authorization check.
+  rpc Check(CheckRequest) returns (CheckResponse);
+}
+
+// The request for a CheckService.Check rpc.
+// Checks whether a specific subject is related to an object.
+message CheckRequest {
+  // The related object in this check.
+  Object object = 1;
+  // The relation this check.
+  string relation = 2;
+  // The concrete subject id to check.
+  string subject_id = 3;
+  // Optional. The staleness bound of this check.
+  //
+  // It specifies the clients last known snapshot token
+  // that the server uses to only check against ACLs newer
+  // than specified token snapshot in order to compute the
+  // result of this check.
+  //
+  // Leave this field blank if...
+  //  - your application strictly requires to act on up-to-date data
```

Comment:
From the read-api:

> If the request doesn’t contain a zookie, Zanzibar will choose a reasonably recent snapshot, possibly offering a lower latency response than if a zookie were provided.

Do you think the same applies to check or is a check request without a zookie automatically a "content-change" check that returns the new versions zookie? From my understanding it would be possible to have a check without a zookie that then is evaluated at a "reasonably recent snapshot", but that returns no new zookie. Maybe my understanding is wrong here.

Reply:
Right, I'm about to redesign the CheckRequest a bit.

Reply:
Done